ci/builders/linux_host_engine.json

```diff
@@ -0,0 +1,174 @@
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_debug/zip_archives/",
+                    "include_paths": [
+                        "out/host_debug/zip_archives/linux-x64/artifacts.zip",
+                        "out/host_debug/zip_archives/linux-x64/linux-x64-embedder",
+                        "out/host_debug/zip_archives/linux-x64/font-subset.zip",
+                        "out/host_debug/zip_archives/flutter_patched_sdk.zip",
+                        "out/host_debug/zip_archives/dart-sdk-linux-x64.zip",
+                        "out/host_debug/zip_archives/flutter-web-sdk-linux-x64.zip",
+                        "out/host_debug/zip_archives/linux-x64-flutter-gtk.zip"
+                    ],
+                    "name": "host_debug"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "debug",
+                "--full-dart-sdk",
+                "--prebuilt-dart-sdk",
+                "--build-embedder-examples"
+            ],
+            "name": "host_debug",
+            "ninja": {
+                "config": "host_debug",
+                "targets": [
+                    "flutter/build/archives:artifacts",
+                    "flutter/build/archives:linux-embedder",
+                    "flutter/build/archives:flutter_patched_sdk",
+                    "flutter/build/archives:dart_sdk_archive",
+                    "flutter/build/archives:flutter_web_sdk",
+                    "flutter/tools/font-subset",
+                    "flutter/shell/platform/linux:flutter_gtk"
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_profile/zip_archives/",
+                    "include_paths": [
+                        "out/host_profile/zip_archives/linux-x64-profile/linux-x64-flutter-gtk.zip"
+                    ],
+                    "name": "host_profile"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "profile",
+                "--no-lto",
+                "--prebuilt-dart-sdk",
+                "--build-embedder-examples"
+            ],
+            "name": "host_profile",
+            "ninja": {
+                "config": "host_profile",
+                "targets": [
+                    "flutter",
```

Comment:
@dnfield if we separate unit-tests to their own target we can remove the top level target, what do you think?

Reply:
SGTM

Reply:
Done, created the unittest target and updated this config to use it instead of flutter